src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -61,7 +62,7 @@ function drawTextWithBackground(text: string, x: number, y: number, ctx: CanvasR
   ctx.fillText(text, x, y);
 }
 
-function drawLineOrBarChartValues(
+function drawLineOrBarOrRadarChartValues(
```

Comment:
![pas-bien-joli](https://github.com/user-attachments/assets/0b05d909-62b5-4685-b44f-564e445b556c)

Reply:
At least it says what it does :D 

Maybe `drawValuesAboveDataPoints`  ? And for horizontal charts it'd be `drawValuesLeftOfDataPoints`

Reply:
No it's good like that ;-)